apis/nucleus/src/components/listbox/ListBoxRadioButton.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { makeStyles } from '@nebula.js/ui/theme';
+import { Radio } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  denseRadioButton: {
+    height: '100%',
+    boxSizing: 'border-box',
+    '& svg': {
+      width: '0.7em',
+      height: '0.7em',
+    },
+  },
+  radioButton: {
+    right: '5px',
+  },
+}));
+
+export default function ListBoxRadioButton({ checked, label, dense }) {
+  const styles = useStyles();
+
+  return (
+    <Radio
+      checked={checked}
+      value={label}
+      name={label}
+      inputProps={{ 'aria-labelledby': label }}
+      className={dense ? styles.denseRadioButton : styles.radioButton}
+      style={{ backgroundColor: 'transparent' }}
+      disableRipple
```

Comment:
This could also be set as a global property override in `theme.js` but I think is fine too.